### ISSUE TITLE
DST Staging Review: Indicate the full screen reader only text

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/vets-design-system-documentation#5685]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Design System Staging Information
**Component:** va-card-status
**Staging Review ticket:** [Staging Review: va-card-status](https://github.com/department-of-veterans-affairs/va.gov-team/issues/131892)

## Findings details
**VA.gov Experience Standard - issue:** [User has enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**This is an issue with the:** guidance
**High-priority:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Guidance currently indicates that the text "Status:" is announced for screen readers, but doesn't indicate that the type of status is included in that announcement (i.e. - "Status: info [tag content]". That context may be important for end users to prevent redundant announcements  - i.e. "Status error: Error"

## Recommended action
- Add guidance on the Card Status component to indicate the full text of the screen reader only text so that authors are more informed when they choose their status text

## References

- **Accessibility Defect Severity:** [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow
- **VADS Guidance:** N/A

---

### Next Steps for DST

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Governance help validating the issue, please comment on the ticket
- If this ticket has a `high-priority` label, please address as soon as possible so VFS teams are not impacted
- If this ticket has a `consider` label, please consider for future implementation
